src/spirit-view/spirit-view.html

```diff
@@ -96,6 +96,12 @@
                     }
                 }
                 return true;
+            },
+
+            _removeName: function() {
```

Comment:
So would it be better to set the default name as the placeholder and not change the name if the input is empty or set it as the value and use validation to not let them leave if there is no name in the input?